engine/swarm/admin_guide.md

```diff
@@ -267,11 +296,16 @@ balance across the swarm. When new tasks start, or when a node with running
 tasks becomes unavailable, those tasks are given to less busy nodes. The goal
 is eventual balance, with minimal disruption to the end user.
 
-If you are concerned about an even balance of load and don't mind disrupting
-running tasks, you can force your swarm to re-balance by temporarily scaling
-the service upward.
+In Docker 1.13 and higher, you can use the `--force` or `-f` flag with the
+`docker service update` command to force the service to redistribute its tasks
+across the available worker nodes. This will cause the service tasks to restart.
+Client applications may be disrupted. If you have specified an `--update-delay`
+value for your service, the service will use a rolling restart.
```

Comment:
See comment above. `--update-delay` does not control whether a rolling restart is used, only whether there is a delay between tasks being updated in a rolling update.